WIllJonesUserDataInfoProject/Program.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace WIllJonesUserDataInfoProject
+{
+    class MainClass
+    {
+        public class UserDataInfo
+        {
+            public string FirstName { get; set; }
+            public string LastName { get; set; }
+            public int FavNumber { get; set; }
+            public int NumOfPets { get; set; }
+
+
+            public void GreetingWithUserInfo()
+            {
+                Console.WriteLine("It's nice to learn all these things about you " + FirstName + " " + LastName + "!");
+               
+            }
+        }
+
+        public static void Main()
+        {
+            UserDataInfo user = new UserDataInfo();
+            Console.WriteLine("What is your first name?");
```

Comment:
These questions and answers on lines 26-35 would probably benefit from a blank line separating each pair to improve readability, for example:
```CS
Console.WriteLine("What is your first name?");
user.FirstName = Console.ReadLine();

Console.WriteLine("What is your last name?");
user.LastName = Console.ReadLine();

Console.WriteLine("What is your favorite number?");
int FavNumber = int.Parse(Console.ReadLine());
```